src/porepy/grids/mortar_grid.py

```diff
@@ -1,19 +1,20 @@
 """ Module containing the class for the mortar grid.
 """
 import warnings
+from enum import Enum
 from typing import Dict, Generator, List, Optional, Tuple, Union
 
 import numpy as np
 from scipy import sparse as sps
 
 import porepy as pp
 
-# Module level constants, used to define sides of a mortar grid.
-# This is in essence an Enum, but that led to trouble in pickling a GridBucket.
-NONE_SIDE = 0
-LEFT_SIDE = 1
-RIGHT_SIDE = 2
-WHOLE_SIDE = np.iinfo(type(NONE_SIDE)).max
+
+class MortarSides(Enum):
+    # Enum of constants used to identify the grids on each side of the mortar
+    NONE_SIDE = 0
+    LEFT_SIDE = 1
+    RIGHT_SIDE = 2
```

Comment:
why did you remove the WHOLE_SIDE? We probably never used it, but maybe you've a specific reason

Reply:
1) I did not understand when we need it, so we'll introduce it later if necessary. 
2) If we reintroduce it, it should get a standard number, not some strange trick to get a random number.